grunt-ngdocs-index.tmpl

```diff
@@ -239,6 +239,16 @@
 
         <div id="loading" ng-show="loading">Loading...</div>
 
+        <span ng-show="breadcrumb[breadcrumb.length-1].name === 'API Documentation'">
+          <br/>
+          <h1><strong>Welcome to Angular-PatternFly!</strong><h1>
+          <h2>Angular-PatternFly contains custom AngularJS directives which adhere to the PatternFly Refererence Implementation.</h2>
+          <h2>PatternFly is a community project that promotes design commonality and improved user experience. Its offerings
+          include visual and behavioral patterns based on research and UX best practices that solve common design problems.<h2>
+          <h3>For more information, please see <a href="https://www.patternfly.org/">https://www.patternfly.org</a></h3>
+          <h3>PatternFly and Angular-PatternFly issue tracking is located <a href="https://patternfly.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=PTNFLY&view=planning.nodetail">here</a></h3>
```

Comment:
Is that entirely true since we also use GH issues? Also (nitpick) the lack of a period on this sentence bothers me :smile: